silkworm/core/execution/evm.cpp

```diff
@@ -97,10 +97,16 @@ CallResult EVM::execute(const Transaction& txn, uint64_t gas) noexcept {
 
 evmc::Result EVM::create(const evmc_message& message) noexcept {
     evmc::Result res{EVMC_SUCCESS, message.gas, 0};
+    const evmc_revision rev{revision()};
```

Comment:
Moving this declaration here is not necessary and decreases readability, so I would leave it where it was (see below)

Reply:
Done